examples/device/hid_multipleinterface/src/usb_descriptors.h

```diff
@@ -0,0 +1,34 @@
+/* 
```

Comment:
This file can be removed since it is not needed. Each interface in this example is single report and does not need report id.